org.eclipse.scanning.sequencer/src/org/eclipse/scanning/sequencer/AcquisitionDevice.java

```diff
@@ -300,66 +301,66 @@ public void run(IPosition parent) throws ScanningException, InterruptedException
 
 			RunnableDeviceServiceImpl.setCurrentScanningDevice(this); // Alows Jython to get and pause/seek.
 
-	        // TODO Should we validate the position iterator that all
-	        // the positions are valid before running the scan?
-	        // It was called limit checking in GDA.
-	        // Sometimes logic is needed to implement collision avoidance
+			// TODO Should we validate the position iterator that all
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)